packages/legend-shared/src/network/NetworkUtils.ts

```diff
@@ -493,6 +496,21 @@ export const downloadFile = (
   }
 };
 
+export const readFileAsText = (file: File): Promise<string> =>
```

Comment:
Let's move this out of here. Let's create a new file called BrowserUtils or something